tests/vmi_configuration_test.go

```diff
@@ -1954,17 +1958,22 @@ var _ = Describe("[sig-compute]Configurations", func() {
 			Expect(disks[1].Alias.GetName()).To(Equal("ephemeral-disk2"))
 			Expect(disks[1].Driver.Cache).To(Equal(cacheWritethrough))
 
+			By("checking if requested cache 'writeback' has been set")
+			Expect(disks[2].Alias.GetName()).To(Equal("ephemeral-disk5"))
+			Expect(disks[2].Driver.Cache).To(Equal(cacheWriteback))
```

Comment:
Can you also adjust one migation test to ensure that we can migrate when it is set?

Reply:
+1 to this. That's the only thing I see missing here.

Reply:
Yup, good point. Added.